src/paths/rent-control.yaml

```
@@ -0,0 +1,58 @@
+get:
+  tags:
+    - Encadrement des loyers
+  summary: Récupérer des encadrements des loyers
+  description: >-
+    Retourne les encadrements des loyers correspondant aux critères demandés
```

:magic_wand: **chore**:
```suggestion
    Retourne les encadrements des loyers correspondants aux critères demandés
```